app/Http/Controllers/Twill/BaseApiController.php

```diff
@@ -232,11 +232,40 @@ protected function getIndexTableColumns(): TableColumns
     public function getForm(TwillModelContract $model): Form
     {
         $model->refreshApi();
```

Comment:
The changes in this file are just rearranging the edit page, moving the Datahub and Timestamps fieldsets to the sidebar.